### PR TITLE
Create visualign.flat.jsonld

### PR DESCRIPTION
**Pull request by Majpuc**
This is a new schema for VisuAlign

### CODE REVIEW

**Comment by lzehl** [comment]
> This is a new schema for VisuAlign

@Majpuc thanks. It is a new metadata instance not a schema, though :wink:

File: instances/data/contentTypes/visualign.flat.jsonld [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "@context": {
+      "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/contentTypes/application/vnd.visualign.flat",
```
Comment (lzehl) [inline instances/data/contentTypes/visualign.flat.jsonld]: Could you change this to 
`"@id": "https://openminds.ebrains.eu/instances/contentTypes/application_vnd.visualign.flat"` ?

It is a new convention I want to force for consistency. The existing instances will also change soon to this convention.

File: instances/data/contentTypes/visualign.flat.jsonld [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "@type": "https://openminds.ebrains.eu/core/ContentType",
+    "fileExtension": [
+      ".flat"
+    ],
+    "name": "application/vnd.visualign.flat",
```
Comment (lzehl) [inline instances/data/contentTypes/visualign.flat.jsonld]: For what is this file format good for? and is it based on a generic media type (e.g. JSON or CSV) ?

File: instances/data/contentTypes/visualign.flat.jsonld [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "relatedMediaType": null,
+    "synonym": [
+      "VisuAlign flat file"
+      ]
+  }
+
```
Comment (lzehl) [inline instances/data/contentTypes/visualign.flat.jsonld]: There seems to be a formatting error at the end of the JSONLD. Maybe the newline is missing after the dictionary is defined or there is one newline too much?

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/data/contentTypes/visualign.flat.jsonld (hunk 1 above)
Comment (Majpuc) [reply instances/data/contentTypes/visualign.flat.jsonld]: yes, I'll change it

File: instances/data/contentTypes/visualign.flat.jsonld (hunk 2 above)
Comment (Majpuc) [reply instances/data/contentTypes/visualign.flat.jsonld]: This is a proprietary file format that contains atlas map information

File: instances/data/contentTypes/visualign.flat.jsonld (hunk 3 above)
Comment (Majpuc) [reply instances/data/contentTypes/visualign.flat.jsonld]: Sorry about that. One line too much!

**Review by lzehl — APPROVED** [review]

**Pull request merged**